### PR TITLE
Add powershell scripts for running the backend on windows

### LINKED ISSUE #2: Create a api for getting keywords
We need to create a rest api which the front end will use to get the keywords and their description.

### PR DESCRIPTION
**Pull request by annk15**
Added powershell scripts for running the backend components on windows.
Also fixed some minor bugs and added support for a new "url" column in the database which refer to the source wiki page.

Closes #2

### CODE REVIEW

File: backend/database/run.ps1
Comment (OscarLarm) [inline backend/database/run.ps1:1]: We could make the names different for .\backend\database\run .ps1 and .\backend\server\run .ps1 to make it clearer what each one does

**Comment by git-hub-oscar** [comment]
Looks great! There's just some small clarification between file names I think could make the repo easier to read. 
Other than that it looks very good and is ready for merge 🥇

**Review by OscarLarm — CHANGES_REQUESTED** [review]
Looks great! There's just some small clarification between file names I think could make the repo easier to read.
Other than that it looks very good and is ready for merge 🥇

File: backend/database/run.ps1
Comment (annk15) [reply backend/database/run.ps1:1]: Agree, I have updated the file names 👍

**Review by OscarLarm — APPROVED** [review]
Looks great! Ready to merge! 💯 🚀

**Pull request merged**